### PR TITLE
Support React 17

### LINKED ISSUE #22: Support React 17
We shall start support React 17 together with previous version to not block projects willing to update to latest.

### PR DESCRIPTION
**Pull request by nitzanashi**
closes #22

### CODE REVIEW

**Review by dobladov — APPROVED** [review]
Cool, I had no idea this was possible. 
```
^16.8.0 || ^17.0.0
```

**Comment by nitzanashi** [comment]
> Cool, I had no idea this was possible.
> 
> ```
> ^16.8.0 || ^17.0.0
> ```

me neither, @godban did

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## Unreleased
+
+Support react 17
```
Comment (godban) [inline CHANGELOG.md:8]:
```suggestion
Support React 17
```

**Comment by nitzanashi** [comment]
https://codesandbox.io/s/black-wood-w5ohp if you wanna check it

**Review by godban — APPROVED** [review]

**Pull request merged**